### PR TITLE
Feature/account fiscal position rule purchase migration 8.0

### PR DESCRIPTION
**Pull request by renatonlima**

### CODE REVIEW

**Comment by rvalyi** [comment]
just rebased on 8.0 so tests are passing again. Need review now.

**Comment by rvalyi** [comment]
:+1:

**Comment by rvalyi** [comment]
ping @mileo @danimaribeiro time for a review?

**Comment by mileo** [comment]
LGTM :+1:

File: account_fiscal_position_rule_purchase/purchase_view.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	<data>
+
+		<record id="account_fiscal_position_rule_purchase_order_form" model="ir.ui.view">
+			<field name="name">account.fiscal.position.rule.purchase.order.form</field>
+			<field name="model">purchase.order</field>
+			<field name="inherit_id" ref="purchase.purchase_order_form"/>
+			<field name="arch" type="xml">
+				<field name="partner_id" position="attributes" >
+					<attribute name="context_fiscalrules">{'company_id': company_id}</attribute>
+				</field>
```
Comment (danimaribeiro) [inline account_fiscal_position_rule_purchase/purchase_view.xml]: web_context_tunnel is needed in the depends.

**Comment by danimaribeiro** [comment]
You could add view and model folders, and remove the file account_fiscal_position_rule_purchase_security.xml which is empty.

The rules are working as expected after adding web_context_tunnel as dependency.

File: account_fiscal_position_rule_purchase/purchase_view.xml (hunk 1 above)
Comment (mileo) [inline account_fiscal_position_rule_purchase/purchase_view.xml]: :+1:

**Comment by rvalyi** [comment]
@danimaribeiro took your considerations into account. Good catch about the missing web_context_tunnel dependency. So merging now as the module was approved otherwise.

**Pull request merged**